agent/task.go

```diff
@@ -249,19 +241,19 @@ func (tc *taskContext) hadTimedOut() bool {
 // makeTaskConfig fetches task configuration data required to run the task from the API server.
 func (a *Agent) makeTaskConfig(ctx context.Context, tc *taskContext) (*model.TaskConfig, error) {
 	if tc.project == nil && tc.version == nil {
-		tc.logger.Execution().Info("Fetching project config.")
+		grip.Info("Fetching project config.")
```

Comment:
can we avoid the grip calls? they don't make a lot of sense in the context of task execution in the agent.

Reply:
what should we do instead?

Reply:
Calling grip logs to a file on the host running the agent, which basically no one will ever see. I think logging to that is only useful if there is no task to log to. Otherwise you should log to the task like in the before diff. The messages in this function don't feel very important though so I think we should just not log them